### PR TITLE
Freaks list

### PR DESCRIPTION
**Pull request by cipry13**
#### Description

Setup UI and freaks list in the Freaks Screen

### CODE REVIEW

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/Freak.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    val level: String,
+    val description: String,
+    var image: Int,
+    val skills: List<String>,
+    val projects: List<String>
+) : Serializable {
+    // Function for better readability
+    fun printFreak(): String {
+        return "$firstName $lastName \n$role : $norm, $level \n$description \nskills: $skills \nprojects: $projects"
+    }
```
Comment (DeeaMatei) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/Freak.kt]: This should display the elements from the lists Skills and Projects, not the whole list like "[iOS, Android]"

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/ItemAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+     * Passes a [Freak] object with the new Intent
+     */
+    override fun onBindViewHolder(holder: ItemViewHolder, position: Int) {
+        val item = freaksList[position]
+        item.image = R.drawable.testimage
+        holder.textView.text = item.firstName
+        holder.imageView.setImageResource(item.image)
+
+        // Used for debug purposes
+        Log.d("Testing", "$position $item");
```
Comment (DeeaMatei) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/ItemAdapter.kt]: Remove this kind of testing

**Review by DeeaMatei — CHANGES_REQUESTED** [review]
:gem: Looking good, just a few changes needed

**Review by DeeaMatei — APPROVED** [review]
:rice_ball: Nice

File: features/freaks/src/main/res/layout/list_item.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<com.google.android.material.card.MaterialCardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="140dp"
+    android:layout_height="140dp"
+    android:layout_marginStart="24dp"
+    android:layout_marginEnd="24dp"
+    android:layout_marginTop="24dp"
+    android:background="@drawable/customborder"
+    app:cardCornerRadius="8dp"
```
Comment (srobert99) [inline features/freaks/src/main/res/layout/list_item.xml:11]: Hard coded values

**Review by srobert99 — APPROVED** [review]
very good

**Comment by cipry13** [comment]
[Android: Freaks Screen - Setup UI and freaks list](https://trello.com/c/7dXztOPH/37-android-freaks-screen-setup-ui-and-freaks-list)

**Pull request merged**